File: Code Chef/JavaScript/300-399/300 Bull or Bear BULLBEAR.js
process.stdin.setEncoding("utf-8");

const strToInt = (strArray) => {
    const intArray = strArray.map((strItem) => strItem.split(" ").map(Number));
    return intArray;
};

const findProfitOrLoss = (buyingPrice, sellingPrice) => {
    (sellingPrice === buyingPrice)
        ? console.log("NEUTRAL")
        : (sellingPrice > buyingPrice)
            ? console.log("PROFIT")
            : console.log("LOSS");
}

process.stdin.on("data", (input) => {
    const testcases = strToInt(input.trim().split("\n").slice(1));
    
    testcases.forEach((deal) => {
        const [buyingPrice, sellingPrice] = deal;
        findProfitOrLoss(buyingPrice, sellingPrice);
    });
});

File: Code Chef/C Programming/300-399/300 Bull or Bear BULLBEAR.c
#include <stdio.h>
int main(void)
{
	int t,x,y;
	scanf("%d",&t);
	for(int i=0;i<t;i++)
	{
	    scanf("%d %d",&x,&y);
	    if(x==y)
	    {
	        printf("NEUTRAL\n");
	    }
	    else if(x<y)
	    {
	        printf("PROFIT\n");
	    }
	    else
	    {
	        printf("LOSS\n");
	    }
	}
	return 0;
}